python/mxnet/contrib/onnx/_import/op_translations.py

```diff
@@ -43,32 +43,42 @@ def add(attrs, inputs, cls):
     """Adding two tensors"""
     new_attr = {}
     if 'broadcast' in attrs and attrs['broadcast'] == 1:
-        op_value = translation_utils._fix_bias_shape('broadcast_add', inputs, cls)
+        broadcast_axis = attrs['axis']
```

Comment:
Won't block here, but add, subtract, multiply divide functions code can be reused.